doc-templates/sandbox/MapboxVectorTilesApi.md

```diff
@@ -103,6 +103,15 @@ The feature must be configured in `router-config.json` as follows
         "minZoom": 14,
         "cacheMaxSeconds": 60
       },
+      // Contains just stops and real-time information for them
+      {
+        "name": "realtimeStops",
+        "type": "Stop",
+        "mapper": "DigitransitRealtime",
+        "maxZoom": 20,
+        "minZoom": 14,
+        "cacheMaxSeconds": 600
```

Comment:
For the other realtime updaters, we have suggested 60 second cache time. Not sure what is actually optimal here but we could use the same.